### ISSUE TITLE
ProfileBannerUrl is not returning the banner instead it is returing background image

### ISSUE THREAD

**Issue by ashGHub** [linvi/tweetinvi#329]

**Comment by linvi** [comment]
I will look into this. Thank you for reporting it.

**Comment by linvi** [comment]
I have verified and it works properly for me. 
Please ensure that you are using version 1.0.

**Comment by linvi** [comment]
Please reply before the end of the weekend as I will send the new version 1.1 probably this weekend.

**Comment by ashGHub** [comment]
Yes you are right. I was using version 0.9.13. After updating to version 1.0 it worked properly. Sorry for wasting your time. I should've checked it first by my self.
